src/components /Calendar.jsx

```diff
@@ -0,0 +1,202 @@
+import React, { useState, useEffect } from "react";
+
+import { Calendar, momentLocalizer } from "react-big-calendar";
+import moment from "moment";
+import "react-big-calendar/lib/css/react-big-calendar.css";
+
+import { Fragment, useRef } from "react";
+import { Dialog, Transition } from "@headlessui/react";
+import { ExclamationTriangleIcon } from "@heroicons/react/24/outline";
+
+import uuid from "react-uuid";
+
+const CalendarComponent = (props) => {
+  // console.log(props);
+  const data = props.medicineData;
```

Comment:
I think we should rename this to medicineData
```suggestion
  const { medicineData } = props
```